### PR TITLE
Provide help on CORS script errors

### PR DESCRIPTION
**Pull request by KamilSzostak**

### CODE REVIEW

File: JavaScript/JavaScriptSDK/Util.ts [hunk 1]
```diff
@@ -470,7 +470,7 @@ module Microsoft.ApplicationInsights {
         * happens in a script from other domain (cross origin, CORS).		
         */
         public static isCrossOriginError(message: string, url: string, lineNumber: number, columnNumber: number, error: Error): boolean {
-            return (message === "Script error." || message === "Script error") && error === null;
+            return (message === "Script error." || message === "Script error") && error == null;
```
Comment (OsvaldoRosado) [inline JavaScript/JavaScriptSDK/Util.ts]: Why is this no longer triple-equals?

Comment (KamilSzostak) [inline JavaScript/JavaScriptSDK/Util.ts]: IE is using `undefined`, all other browsers `null`. This will match both values.

Comment (OsvaldoRosado) [inline JavaScript/JavaScriptSDK/Util.ts]: Would `!error` be better so that linters don't pick this up in the future?

Comment (OsvaldoRosado) [inline JavaScript/JavaScriptSDK/Util.ts]: Also curious. Does this string change in different locales?

Comment (KamilSzostak) [inline JavaScript/JavaScriptSDK/Util.ts]: I changed it to `!error`. I checked in polish version of Firefox and the error message is not localized.

**Pull request merged**